src/server/README.md

```diff
@@ -56,13 +61,17 @@ myServer
 ```
 
 ## Configuration
-If you have an environment variable called `PORT`, your server will start on this port, otherwise it will default to `3000`. If you want to force a `PORT`, please include it in the initialization as follows:
+
+On VIP Go servers, we use an environment variable called `PORT` to start up the server, which is the default that must be used. If not defined, the port defaults to `3000`. If you want to force a `PORT`, please include it in the initialization as follows:
+
 ``` js
-const myServer = server( app, { PORT: 8000 } );
+const myServer = server( app, { PORT: process.env.PORT || 8000 } );
```

Comment:
Should we use `process.env.PORT` here ? The order in `./src/server/index.js` is: `PORT || process.env.PORT || 3000`

Reply:
I think this is fine for now.

Although, if we want to simplify and not have clients worry about this implementation detail, we can always default to `process.env.PORT` in `./src/server/index.js` (i.e. `process.env.PORT || PORT || 3000`). The only downside is that it would be a bit unexpected in a local environment where someone is passing in a custom port ad also happens to have `process.env.PORT` set (although, that feels like an edge case).

Reply:
Going to merge this for now, and we can revisit the PORT discussion if we find it confusing or a problem.